build.gradle

```diff
@@ -66,3 +67,37 @@ tasks.register('copyDockerCompose', Copy) {
 
 }
 tasks.compileJava.dependsOn('copyDockerCompose')
+
+jacocoTestReport {
+    dependsOn(integrationTest)
```

Comment:
Maybe here a dependency on the unit tests should be added.

Reply:
Done